### ISSUE TITLE
My profile photo cannot be displayed in full size

### ISSUE THREAD

**Issue by r0polach** [signalapp/Signal-Android#11033]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description

While my contacts' profile photos I can display in full size (i.e. not in small circle),
the same I cannot do with my own profile photo (while my contacts can diaplay
*my* photo in full size)

### Steps to reproduce
- go to app settings
- click the first item (my photo in smallest circle and phone number)
- click my photo in small circle
- 

**Actual result:** There is no item to view it in full size, only items for taking new photo, select new photo and delete the photo.
**Expected result:** There should be an item to view full size photo, because even my contacts can display *my* photo in full size.

**Comment by Anik-Bardhan** [comment]
Can I work on this issue?

**Comment by r0polach** [comment]
Yes, this is still relevant. I have no idea, if something blocking it.

**Comment by r0polach** [comment]
This is still relevant.
I have no idea what's blocking it.

**Comment by r0polach** [comment]
Thanks! It works - in the last released version.
